### ISSUE TITLE
Incorporate Shortcut Links as Customization Options

### ISSUE THREAD

**Issue by g4rr3tt** [WTFender/aws-sso-extender#141]
### Description

With the recent addition of shortcut links in AWS IAM Identity Center ([announcement](https://aws.amazon.com/about-aws/whats-new/2024/04/aws-iam-identity-center-shortcut-links-aws-access-portal/)), it would be great to incorporate those as customization options, similar to how colors can be set per profile.

As an example, this could be used to always go to a specific region for a profile, regardless of the last one used.

It could also be used to go to a specific section of the console. If you were a member of the Networking team, for instance, you could set each profile to go to the VPC dashboard.

**Comment by WTFender** [comment]
Good idea. I'm thinking a global "default page" and "default region" setting would be useful, then the ability to set a preferred page and region on each profile.

To your point, NetworkOperation role likely wants the VPC page as their default, but may want home/IAM/whatever for everything else.
